fiftyone/core/odm/database.py

```diff
@@ -1542,6 +1542,90 @@ def delete_runs(name, dry_run=False):
     )
 
 
+def get_indexed_values(
+    collection,
+    field_or_fields,
+    *,
+    index_key=None,
+    query=None,
+    values_only=False,
+):
+    """Returns the values of the field(s) for all samples in the given collection
+    that are covered by the index. Raises an error if the field is not indexed.
+
+    Args:
+        collection: a ``pymongo.collection.Collection`` or
+            ``motor.motor_asyncio.AsyncIOMotorCollection``
+        field_or_fields (str or list): the field name(s) to retrieve.
```

Comment:
I know its annoying, but our docstring style is to use `()` to denote default values, not types. So I'd remove `(str or list)` here.

Note that `(None)` and `(False)` below are default values, not types 😄